### PR TITLE
Changes to make it compile on Macos (with additonal library installed)

### PR DESCRIPTION
**Pull request by joaolsd**
Got it to compile (and run) on macOS with these changes

### CODE REVIEW

**Comment by berthubert** [comment]
wouldn't this break installations where the library is not in /usr/local/lib ?

**Comment by joaolsd** [comment]
It would not compile of it wasn’t there. On the other hand if you change the default you would be aware and changes this accordingly. 
A perfect solution would look for it wherever it was but I didn’t get that far. 80/20 rule

Joao

File: README.md [hunk 1]
```diff
@@ -33,3 +33,9 @@ support page](https://support.google.com/a/answer/10026322?hl=en).
 
 Note that this splits out Google services and Google cloud user IP
 addresses.
+
+## Macos
+You will need to install the pcaudio library:
+  https://github.com/espeak-ng/pcaudiolib#mac-os
+In turn this needs automake installed (from xcode or brew). With that it is standard install
+WIth libpcaudio in place, follow instructions above.
```
Comment (hextriclosan) [inline README.md:41]: W**i**th

**Comment by M-Amrollahi** [comment]
> Changes to make it compile on Macos (with additonal libraru installed)

Hello, may I know, which package should I install through brew? I do not have XCode.
Brew does not find a package ```libpcaudio```.

**Comment by joaolsd** [comment]
> On 23 Aug 2022, at 06:19, Mahdi Amrollahi ***@***.***> wrote:
> 
> ﻿
> Changes to make it compile on Macos (with additonal libraru installed)
> 
> Hello, may I know, which package should I install through brew? I do not have XCode.
> 

Automake
> Brew does not find a package libpcaudio.
> 
With automake installed (brew) you can install libpcaudio from the given url

**Pull request closed without merging**